go/pkg/bertymessenger/handlers.go

```diff
@@ -1201,3 +1202,47 @@ func (h *eventHandler) indexMessage(tx *dbWrapper, id string, am *mt.AppMessage)
 
 	return nil
 }
+
+func (h *eventHandler) handleAppMessageSetGroupInfo(tx *dbWrapper, i *mt.Interaction, amPayload proto.Message) (*mt.Interaction, bool, error) {
+	payload := amPayload.(*mt.AppMessage_SetGroupInfo)
+
+	if i.GetConversation().GetType() == mt.Conversation_MultiMemberType {
+		if i.GetIsMine() {
+			return i, false, nil
+		}
+
+		cpk := i.GetConversationPublicKey()
+		c, err := tx.getConversationByPK(cpk)
+		if err != nil {
+			return nil, false, err
+		}
+
+		if c.GetCreatedDate() > i.GetSentDate() {
```

Comment:
```suggestion
		if c.GetInfoDate() > i.GetSentDate() {
```
you will need to add `info_date` to the Conversation model